### ISSUE TITLE
what is cd

### ISSUE THREAD

**Issue by Darkingg12** [salmonslay/msp-tools#1]
I'm a novice at these things
Please tell me what's going on with this?

cd msp-tools
npm install

**Comment by Darkingg12** [comment]
I downloaded everything,
I added information about my character,
I don't know how to run npm, is it some program or what

**Comment by Darkingg12** [comment]
<img width="522" alt="Przechwytywad1nie" src="https://github.com/salmonslay/msp-tools/assets/114777215/db9a4f0e-4b0a-4dfd-9815-800c62a8e19f">

**Comment by Darkingg12** [comment]
<img width="617" alt="Przechwytywane32ie" src="https://github.com/salmonslay/msp-tools/assets/114777215/0b0d57ab-2349-442d-bc79-d68f279dd5f3">








Finally this

**Comment by Darkingg12** [comment]
My problem is probably that it doesn't read my login details

I don't know what I did wrong, does the file name matter?

**Comment by salmonslay** [comment]
Yes, the credentials should be stored in a file called `.env` following the same structure as the example provided.

**Comment by Darkingg12** [comment]
does it only work on msp1 or also on msp2?

**Comment by Darkingg12** [comment]
how should I name the file ".env" without a name, the format itself...

**Comment by salmonslay** [comment]
No, the name needs to be `.env`, not `example.env`. You can just rename it in the operating system you're using - if you aren't sure how to do that you'll have to search it up.

**Comment by Darkingg12** [comment]
Ok, done, but new error 

<img width="728" alt="Przedsadaschwytywanie" src="https://github.com/salmonslay/msp-tools/assets/114777215/6aad8715-485e-4839-94ea-789446d5ee6d">

**Comment by Darkingg12** [comment]
I asked everywhere, they suggest I contact the author about this...

**Comment by salmonslay** [comment]
Seems like they've updated their token endpoint. I got it to work by replacing `eu.mspapis.com` with `eu-secure.mspapis.com` at line 38 and 60 in the file `index.js`. 

![image](https://github.com/salmonslay/msp-tools/assets/47401343/b893ac71-87ad-464d-b43a-31d9778a586f)

I can't really help you any more than that as this is no longer maintained though.

**Comment by Darkingg12** [comment]
so if I edit these lines it will be correct?

**Comment by Darkingg12** [comment]
what if I get this error?

I entered everything correctly...

<img width="519" alt="Przechwdsaytywanie" src="https://github.com/salmonslay/msp-tools/assets/114777215/f4077272-6758-4856-89d9-dd9a41aa933c">


Is it supposed to be given this way?

or use some symbol or space before

<img width="576" alt="Przechwytywdsaanie" src="https://github.com/salmonslay/msp-tools/assets/114777215/ba6a299c-eea1-4b89-81f5-95fd4cfb69d6">


Is it just that the script won't handle it anymore and I should give up?
If ye - how could you have succeeded?

**Comment by salmonslay** [comment]
Your server is incorrect, read the comment right above it (line 14). You should only include the country code and not the full URL.

**Comment by Darkingg12** [comment]
finally it worked, thank you very much for your help!!

but..

does it even send autographs?
it's still standing...


<img width="660" alt="Przechwytywan3213211ie" src="https://github.com/salmonslay/msp-tools/assets/114777215/2b2cde9a-4a8e-4d18-8aec-9ddb7311a198">
